app/views/foreman_plugin_template/layouts/react.html.erb

```diff
@@ -0,0 +1,16 @@
+ 
+<% content_for(:javascripts) do %>
+  <%= webpacked_plugins_js_for :'foreman-tasks' %>
```

Comment:
`webpacked_plugins_js_for :foreman_plugin_template`? Same for the css helper below.